docs/application/dotnet/guides/security/tee-client.md

```diff
@@ -1,5 +1,7 @@
 # TEE Communication
 
+> **Note**   
```

Comment:
Change the Note style
https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md
> [!NOTE]
> `Tizen.Security.TEEC` is deprecated since Tizen 6.5 and will be removed after two releases without any alternatives.

Reply:
Done.